language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -582,12 +581,12 @@ public static List<LSCompletionItem> getRecordFieldCompletionItems(BallerinaComp
      *
      * @param context Language Server Operation Context
      * @param fields  A non empty map of fields
-     * @param symbol  Pair of Raw TypeSymbol and broader TypeSymbol
+     * @param wrapper  Pair of Raw TypeSymbol and broader TypeSymbol
      * @return {@link Optional}   Completion Item to fill all the options
      */
     public static Optional<LSCompletionItem> getFillAllStructFieldsItem(BallerinaCompletionContext context,
```

Comment:
Shall we change the name of the method to `getFillAllRecordFieldsCompletionItem`